### PR TITLE
WIP: add a compose file for local development

### PR DESCRIPTION
**Pull request by sthaha**
WIP: Make it easy to get kepler up and running locally in a container using `compose.yaml`

### CODE REVIEW

**Comment by sthaha** [comment]
cc: @husky-parul @vimalk78 @vimalk78 @KaiyiLiu1234

File: compose.yaml
Comment (vimalk78) [inline compose.yaml:1]: move it under `hack` ?

Comment (sthaha) [reply compose.yaml:1]: +1

File: docker-compose/grafana/dashboards/kepler-dashboard.json
Comment (vprashar2929) [inline docker-compose/grafana/dashboards/kepler-dashboard.json:1]: I think this JSON is from `grafana-dashboards/legacy` which uses old queries. Can we use the latest one for now? Once this PR merges we can enhance it a little more

Comment (sthaha) [reply docker-compose/grafana/dashboards/kepler-dashboard.json:1]: Absolutely!

**Comment by sthaha** [comment]
replaced by #1342

**Pull request closed without merging**